lib/DDG/Spice/Cryptocurrency.pm

```diff
@@ -5,6 +5,8 @@ package DDG::Spice::Cryptocurrency;
 use DDG::Spice;
 with 'DDG::SpiceRole::NumberStyler';
 
+use Data::Dumper;
```

Comment:
not sure if you want to remove this from debugging, but not really doing any harm.